manifests/m/Microsoft/VCRedist/2015+/x64/14.34.31931/Microsoft.VCRedist.2015+.x64.yaml

```
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.VCRedist.2015+.x64
+PackageVersion: 14.34.31931

```

```suggestion
PackageVersion: 14.34.31931.0
```
Must match Control Panel.